patroni/postgresql/__init__.py

```diff
@@ -407,6 +407,14 @@ def start(self, timeout=None, task=None, block_callbacks=False, role=None):
         self._pending_restart = False
 
         configuration = self.config.effective_configuration
+        if "unix_socket_directories" in configuration:
```

Comment:
Actually we should do the same for [stats_temp_directory](https://github.com/zalando/patroni/issues/863)